### ISSUE TITLE
Warnings and Notices in Sofawiki 3.7.3 

### ISSUE THREAD

**Issue by GaneshKandu** [bellenuit/sofawiki#22]
Hi,

After install i am getting Warnings and Notices even ```display_error``` and ```error_reporting``` is off

#### List > Images

```Notice: Trying to get property of non-object in inc/relation.php on line 1181

Warning: Invalid argument supplied for foreach() in inc/relation.php on line 1181

Notice: Trying to get property of non-object in inc/relation.php on line 1181

Warning: Invalid argument supplied for foreach() in inc/relation.php on line 1181

Notice: Trying to get property of non-object in inc/relation.php on line 1181

Warning: Invalid argument supplied for foreach() in inc/relation.php on line 1181

Notice: Trying to get property of non-object in inc/relation.php on line 1181

Warning: Invalid argument supplied for foreach() in inc/relation.php on line 1181
```

#### List > Orphaned Page

```Warning: preg_match(): Unknown modifier '/' in inc/expressionfunction.php on line 1663```

**Comment by bellenuit** [comment]
Hi, it is fixed in 3.7.4

**Comment by GaneshKandu** [comment]
> Hi, it is fixed in 3.7.4

Thanks !
